disnake/abc.py

```diff
@@ -978,10 +978,13 @@ async def _clone_impl(
         base_attrs: Dict[str, Any],
         *,
         name: Optional[str] = None,
+        category_id: Optional[Snowflake] = MISSING,
         reason: Optional[str] = None,
     ) -> Self:
         base_attrs["permission_overwrites"] = [x._asdict() for x in self._overwrites]
-        base_attrs["parent_id"] = self.category_id
+        base_attrs["parent_id"] = (
+            category_id.id if category_id is not MISSING and category_id else self.category_id
```

Comment:
passing `category=None` is valid, but this would then use the existing channel's category instead of setting `parent_id: null`